src/components/reusable/nav/mod.rs

```diff
@@ -107,11 +110,14 @@ pub fn Nav(cx: Scope<Props>) -> Element {
                             reqCount.with_mut(|count| *count -= 1);
                         }
                     }
-                    MultiPassEventKind::FriendAdded { .. } => {
+                    MultiPassEventKind::FriendAdded { did } => {
                         log::debug!("updating friend request count");
                         if *(reqCount.get()) != 0 {
                             reqCount.with_mut(|count| *count -= 1);
                         }
+                        log::debug!("creating chat");
+                        let _result =
+                            warp::async_block_in_place_uncheck(rg.create_conversation(&did));
```

Comment:
Since this is inside of a use_future, you can remove the `warp::async_block_in_place_uncheck` and just do `rg.create_conversation(&did).await`. I would also suggest checking for any errors (similar to what was being done previously) and log them.

Reply:
Great point!